apps/go-mod-validator/test/e2e.test.ts

```diff
@@ -0,0 +1,81 @@
+import { describe, it, expect } from "vitest";
+import { run } from "../src/go-mod-validator";
+import { join } from "path";
+
+const testDataDir = "./test/data";
+
+function setup(repoName: string) {
+  process.env["INPUT_GO-MOD-DIR"] = join(testDataDir, repoName);
+  process.env["INPUT_GITHUB-TOKEN"] = process.env.GITHUB_TOKEN;
+  process.env["INPUT_DEP-PREFIX"] = "github.com/smartcontractkit";
+  process.env["GITHUB_STEP_SUMMARY"] = "/dev/null";
+}
+
+describe("e2e tests", () => {
```

Comment:
These tests are only validating the binary validation result. Should there be better validation for the individual errors?

Reply:
I tried to keep it pretty KISS as they're e2e tests, we can add more complex ones later on if individual unit tests dont suffice